producer/start-producer.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# set image and container names
+IMAGE_NAME="kafka-producer"
+CONTAINER_NAME="kafka-producer-container"
+
+# Check if the Docker image already exists
+if [[ "$(docker images -q $IMAGE_NAME 2> /dev/null)" == "" ]]; then
```

Comment:
this option could be managed by a flag to allow to rebuild the image

Reply:
done